sources/install/package_c2.sh

```diff
@@ -43,10 +43,13 @@ function install_routersploit() {
 function install_sliver() {
     # CODE-CHECK-WHITELIST=add-aliases
     colorecho "Installing Sliver"
-    git -C /opt/tools/ clone --depth 1 https://github.com/BishopFox/sliver.git
+    # Deletion of --depth 1 due to installation of stable branch
+    git -C /opt/tools/ clone https://github.com/BishopFox/sliver.git
     cd /opt/tools/sliver
-    make
-    mv sliver-* /opt/tools/bin
+    git checkout tags/v1.5.39
```

Comment:
Do we really want to have a static version? Or is it because there's a temporary fix to something?

Reply:
I have an error when running the binary on the master branch and the doc says that to install from build it's best to use a version with a release tag.

Reply:
Ok we need to make it a temporary thing then (see [temporary fix](https://exegol.readthedocs.io/en/dev/community/contributors.html#temporary-fixing-a-tool)), so that we get a reminder to update the version regularly